### PR TITLE
Content added v1

### PR DESCRIPTION
**Pull request by ChrisDesignOps**
First content added in ES

### CODE REVIEW

**Review by andresin87 — CHANGES_REQUESTED** [review]
.

**Comment by jcatdom** [comment]
🥳

File: Atomic-design.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<br/>Son la base de nuestro sistema de diseño y nos permiten agilizar la creación de nuestros productos.
+
+### Organismos
+Son componentes relativamente complejos compuestos de moléculas, y/o átomos y/u otros organismos. (p.e: nestedCheckboxes, Header..). Permiten formar diferentes secciones de una interfaz y definir interacciones claves de nuestros productos.
+
+ℹ️ Si necesitas generar una nueva variable de cualquier componente, tendrás que verificar si lo que necesitas modificar es el organismo, las moléculas que lo componen o sus átomos. 
+<br/>Más información sobre [cómo iterar un componente](https://github.com/turolopezsanabria/design-systems-playbook/blob/master/SUI-Components.md).
+
+### Templates
+Gracias a nuestro sistema átomico (átomos, moléculas y organismos) podemos generar templates de pantallas que definan nuestros productos de manera consistente.
```
Comment (turolopezsanabria) [inline Atomic-design.md:29]:
```suggestion
Gracias a nuestro sistema atómico (átomos, moléculas y organismos) podemos generar templates de pantallas que definan nuestros productos de manera consistente.

```

**Review by turolopezsanabria — CHANGES_REQUESTED** [review]
test

File: Atomic-design.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Atomic design
+
+Nuestras librerías de diseño se basan en un sistema atómico en el que contemplamos diferentes niveles en sus componetizació. Esto nos permite tener un sistema reutilizable, escalable y que nos proporciona consistencia en nuestros productos.
```
Comment (turolopezsanabria) [inline Atomic-design.md:3]:
```suggestion
Nuestras librerías de diseño se basan en un sistema atómico en el que contemplamos diferentes niveles en sus componetización. Esto nos permite tener un sistema reutilizable, escalable y que nos proporciona consistencia en nuestros productos. 

```

**Review by turolopezsanabria — CHANGES_REQUESTED** [review]

**Pull request merged**